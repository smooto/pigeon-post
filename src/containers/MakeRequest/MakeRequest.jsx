import React from 'react';

const MakeRequest = () => (
  <section>
    <h2>New Request</h2>
    <form>
      <input type="url" />
      <input type="button" value="GET" />
      <input type="button" value="POST" />
      <input type="button" value="PUT" />
      <input type="button" value="DELETE" />
      <input type="submit" />
    </form>
  </section>
);

export default MakeRequest;
