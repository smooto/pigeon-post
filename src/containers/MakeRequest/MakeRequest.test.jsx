import React from 'react';
import { shallow } from 'enzyme';
import MakeRequest from './MakeRequest';

describe('MakeRequest component', () => {
  it('renders MakeRequest', () => {
    const wrapper = shallow(<MakeRequest passData={() => {}} passRequest={() => {}} />);
    expect(wrapper).toMatchSnapshot();
  });
});
