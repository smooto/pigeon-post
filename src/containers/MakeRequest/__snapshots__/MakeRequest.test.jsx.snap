// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MakeRequest component renders MakeRequest 1`] = `
<section>
  <h2>
    New Request
  </h2>
  <RequestBar
    onBodyChange={[Function]}
    onSubmit={[Function]}
    onTypeChange={[Function]}
    onUrlChange={[Function]}
  />
</section>
`;
